perceval/simulators/noisy_sampling_simulator.py

```diff
@@ -216,6 +207,8 @@ def _noisy_sampling(
             provider: SamplesProvider,
             max_samples: int,
             max_shots: int,
+            detectors: list[IDetector],
+            detection_type: DetectionType,
```

Comment:
Seems like detection_type is not used.
Also, why do we give this as argument and not in a function like `set_circuit` and `set_selection`?